features/sharding/core/src/main/java/org/apache/shardingsphere/sharding/merge/dal/show/ShowTableStatusMergedResult.java

```diff
@@ -74,10 +74,16 @@ private void merge(final MemoryQueryResultRow row, final MemoryQueryResultRow ne
     }
     
     private BigInteger sum(final Object num1, final Object num2) {
+        if (num1 == null || num2 == null){
+            return BigInteger.ZERO;
```

Comment:
Maybe it should return a non-null num instead of returning zero directly.

Reply:
Uh... it should return null， I’ve already made the modification.

Reply:
Both are null, return null, one is not null, return the non-null one. Is it more appropriate?

Reply:
if the bug  is caused by the view , then is  wouldn't  be a case where on is null and then other is not . but ,it is indeed not elegant !   changed.